src/lib/observe.ts

```diff
@@ -14,17 +14,19 @@
  * limitations under the License.
  */
 
-import {FirstInputPolyfillEntry, NavigationTimingPolyfillEntry} from '../types.js';
-
+import {
+  FirstInputPolyfillEntry,
+  NavigationTimingPolyfillEntry,
+} from '../types.js';
 
 interface PerformanceEntryMap {
-  'event': PerformanceEventTiming[];
-  'paint': PerformancePaintTiming[];
+  event: PerformanceEventTiming[];
```

Comment:
These should stay quoted. I also updated the prettier options to preserve object property quote usage.